src/exo/query_asts.py

```diff
@@ -97,6 +97,9 @@ def __init__(self):
 class R(Type):
     pass
 
+@_dataclass
```

Comment:
@adcastel This is a duplication of the code below

Reply:
Fixed